dcicutils/s3_utils.py

```diff
@@ -220,8 +225,9 @@ def unzip_s3_to_s3(self, zipped_s3key, dest_dir, acl=None):
                 # files twice that we want to further interogate
                 the_file = zipstream.open(file_name, 'r').read()
                 file_to_find = file_name.split('/')[-1]
```

Comment:
This line is equivalent to, and better written as:
```
file_to_find = os.path.basename(file_name)
```
As an example:
```
>>> os.path.basename("foo/bar/baz.html")
'baz.html'
>>> "foo/bar/baz.html".split("/")[-1]
'baz.html'
```
HOWEVER, I don't think the value of this variable is used so I'm not sure why the line is here at all.